cmd/image-processing/main.go

```diff
@@ -171,6 +179,37 @@ func runImageProcessing(ctx context.Context) error {
 		}
 	}
 
+	// check for image vulnerabilities if vulnerability scanning is enabled.
+	var vulns []buildapi.Vulnerability
+	if flagValues.vulnerabilitySettings.Enabled && flagValues.push != "" {
```

Comment:
Where is vulnerability scan done, when the image was already pushed by the build strategy ?

Reply:
I talk with Karan, we need also to ensure that if the `Fail` key is set to true, that only for shp-managed strategies it can work. If is not the case, then a validation should prevent the Build from being created

Reply:
Done.